README.md

```diff
@@ -86,6 +86,28 @@ node {
 For more information about slack messages see [Slack Messages Api](https://api.slack.com/docs/messages)
 and [Slack attachments Api](https://api.slack.com/docs/message-attachments)
 
+# Configuration as code
+
+This plugin supports configuration as code
+Add to your yaml file:
+```yaml
+credentials:
+  system:
+    domainCredentials:
+      - credentials:
+          - string:
+              scope: GLOBAL
+              id: slack-token
+              secret: '${SLACK_TOKEN}'
+              description: Slack token
+
+
+unclassified:
+  slacknotifier:
```

Comment:
Would be nice to add `@Symbol('slackNotifier')` to the descriptor

Reply:
does that need a core bump?
Can't find the annotation.

I had a nightmare of a time trying to bump the core version, due to a test timing issue

Reply:
Added using docs here for 1.x core:
https://wiki.jenkins.io/display/JENKINS/Structs+plugin